paper.tex

```diff
@@ -2056,6 +2003,95 @@ \subsubsection{Future Work}
+
+Initially, we experimented with using a four-column Kanban board to keep track
+of our progress. Figure~\ref{fig:kanbanBoard} shows a screenshot of this page on
+GitHub.
+
+The Kanban technique stems from Toyota in the 1940s. Kanban means ``visual
+signal'' or ``card'' in Japanese, which describes the core component of the
+technique~\autocite{terry2019}. At Toyota, line-workers would move colored cards
+to notify other teams that parts needed for assembly work were in short supply.
+Now, Kanban boards are integral to Agile software development. Still, some teams
+use physical Kanban boards, often using sticky notes and whiteboards.
+
+The advantage of using a Kanban board is that each feature or bug that
+is being worked on can be assigned directly to a collaborator on the project. It
+is then that person's responsibility to work on those tasks and update the
+status of those tasks on the board. We abandoned this when moving onto the main
+project because we came to realize that this structure was not necessary---there
+were only three members on the team, and we were almost always in the same room
+when working on the project. For Editour, we tracked features and issues using
+the Issues page instead, which is discussed in
+Section~\ref{sec:editourFrontendTesting}.
+
+Creating ``pull requests'' through the GitHub web interface has a few advantages
+over directly merging code using git. Firstly, the pull request feature
+integrates with ``Issues'' page very well. By typing `resolves' along with the
+number associated with an issue into the description of a pull request, GitHub
+will automatically close the issue when the pull request is approved. This is
+not only remarkably convenient, but also self-documenting since the descriptions
+of pull requests would contain the issues they were resolving. Secondly,
+creating a pull request allows code to be reviewed by other members of the team
+very easily. Reviewers can view the entire diff and add comments line-by-line.
+Other collaborators can easily respond to those comments. Reviewers can also
+request certain features be changed before the branch is merged, which allowed
+us to enforce that issues with the code were actually addressed.
+% TODO explain what the heck a diff is
+
+% TODO give this paragraph some more context so it doens't seem out of place
+The way we incorporated branches into our workflow helped us collaborate in a
+way that kept us organized. In general, if one of us was working on a specific
+feature, we would create a branch with a name descriptive of that feature. This
+methodology minimized difficult-to-resolve merge conflicts. We modeled this
+workflow off of the ``Git Feature Branch Workflow'' Atlassian
+tutorial~\autocite{atlassian}.
+% TODO explain merge conflicts
+
+Once we deployed Editour to a server so that Atticus could begin to use it,
+we changed the version to v1.0.0 to indicate the first release version. At this
```

Comment:
Explain about semantic versioning here, or at least add a TODO for it

Reply:
wrote a paragraph for that now